supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -256,6 +256,18 @@ The Red{nbsp}Hat Container Catalog no longer exists; it has become part of the R
 
 *See also*: xref:data-grid[Data Grid], xref:red-hat-jboss-data-grid[Red{nbsp}Hat JBoss Data Grid]
 
+[discrete]
```

Comment:
We don't use discrete tags anymore, can you remove this?

```suggestion

```

Reply:
@emmurphy1 I think the only remaining change needed is to remove the tag in line 259. Can you please make this change and we'll try to get this PR approved? Many thanks :)

Reply:
@bredamc done!